apollo-backend/src/main/java/io/logz/apollo/controllers/DeploymentController.java

```diff
@@ -164,4 +166,22 @@ public void cancelDeployment(int id, Req req) {
             lockService.releaseLock(lockName);
         }
     }
+
+    @LoggedIn
+    @POST("/deployment-history")
```

Comment:
You need to add tests to this new endpoint, and remove the unnecessary code that you replaced 💪

Reply:
Thx for all of the help with this! 
done😅 and I'll remove the unnecessary code after I'll merge the new ui.